vignettes/LPWC.Rmd

```diff
@@ -132,6 +132,65 @@ The default is 10.
 Increasing iter increases computational time.
 This is only relevant for the low penalty because the high penalty uses a single C instead of searching over multiple C.
 
+## Interpreting lags
+
+
+```{r, echo = FALSE, fig.height = 2, fig.width=4, warning = FALSE}
+library(ggplot2)
+
+set.seed(29876)
+
+
+a <- rbind(c(rep(0, 5), 8, 0), c(rep(0, 4), 4.3, 0, 0)) + rnorm(2, 0, 0.5)
+
+dat <- data.frame(intensity = as.vector(a), time = rep(c(0, 5, 15, 30, 45, 60, 75), each = 2), genes = factor(rep(c(1, 2), 7)))
+
+a2 <- a
+a2[1, ] <- c(a2[1, 2:7], NA)
+dat2 <- data.frame(intensity = as.vector(a2), time = rep(c(0, 5, 15, 30, 45, 60, 75), each = 2), genes = factor(rep(c(1, 2), 7)))
+
+a3 <- a
+a3[2, ] <- c(NA, a3[2, 1:6])
+a3[1, ] <- c(a3[1, 2:7], NA)
+dat3 <- data.frame(intensity = as.vector(a3), time = rep(c(0, 5, 15, 30, 45, 60, 75), each = 2), genes = factor(rep(c(1, 2), 7)))
+
+
+plot1 <- ggplot(dat, aes(x= time, y = intensity, group = genes)) + geom_line(aes(color = genes), size = 1.5) +  labs(x = "Time (min)") + labs(y = "Intensity") 
+plot1
```

Comment:
When I render the vignette, the figures are left aligned and the tables are center aligned.  Can we center the figures too?

Reply:
Yeah, I spent some time trying to do it but I couldn't figure it out, let me work on that.

Reply:
Fixed